app/models/inventory_item_collection.rb

```diff
@@ -26,9 +26,9 @@ def self.new_from_search_queries(inventory, query_rows)
     inventory.log :info, "#{self} instantiating from queries"
 
     iic = new(:query)
-    query_rows.each_with_index do |query_row, index|
+    query_rows.each_with_index do |query_row, _index|
```

Comment:
there's no reason to use `each_with_index` if we not using the index, can be a simple `each`